srcbook/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "srcbook",
-  "version": "0.0.1-alpha.18",
```

Comment:
Can we keep the alpha here? Are we planning on fixing this as a follow up? I don't like the idea of releasing 0.0.1 by accident

Reply:
I'd prefer to keep alpha for a little longer but I do think we're close to 0.0.1 and then from there a long journey to 1.0.0.

Reply:
Ah yep let me put it back changeset doesn't really work with the alpha stuff since it's not semver compliant